GenderPayGap.WebUI/Views/AdminOrganisationStatus/ViewOrganisationStatus.cshtml

```diff
@@ -0,0 +1,86 @@
+﻿@using GenderPayGap.Core
+@using GenderPayGap.Database
+@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model GenderPayGap.Database.Organisation
+
+@{
+    ViewBag.Title = $"Status history - {Model.OrganisationName} - Administration - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@section BeforeMain {
+    @{
+        var crumbs = new List<CrumbViewModel>
+        {
+            new CrumbViewModel
+            {
+                Text = "Admin",
+                Href = Url.Action("Home", "Admin")
+            },
+            new CrumbViewModel
+            {
+                Text = Model.OrganisationName,
+                Href = Url.Action("ViewOrganisation", "AdminViewOrganisation", new {id = Model.OrganisationId})
+            },
+            new CrumbViewModel
+            {
+                Text = "Status history"
+            }
+        };
+    }
+
+    @(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+    {
+        Crumbs = crumbs
+    }))
+}
+
+<span class="govuk-caption-xl">Administration</span>
+<h1 class="govuk-heading-xl">
+    Status history
+    <br/>
+    <span class="govuk-!-font-size-27">
+        for @(Model.OrganisationName)
+    </span>
+</h1>
+
+@if (Model.Status == OrganisationStatuses.Retired || Model.Status == OrganisationStatuses.Active || Model.Status == OrganisationStatuses.Deleted)
+{
+    @Html.GovUkButton(new ButtonViewModel
+    {
+        Href = Url.Action("ChangeStatusGet", "AdminOrganisationStatus", new {id = Model.OrganisationId}),
+        Text = "Change status"
+    })
+}
+else
+{
+    <span class="govuk-body-m govuk-!-margin-bottom-4">You cannot change status of an organisation that is @(Model.Status)</span>

```

Comment:
It we show this, it might not be clear why we're telling the user that they can't change the status.
It might be nice to put a disabled / unclickable button next to it, to see what they would normally be able to click.
e.g.
```cshtml
<button disabled="disabled" aria-disabled="true" class="govuk-button govuk-button--disabled" data-module="govuk-button" style="vertical-align: baseline;">
    Change status
</button>
<span class="govuk-body" style="color: #626a6e;">
    You cannot change status of an organisation that is @(Model.Status)
</span>
```
**Note:** the `vertical-align: baseline;` lines them up nicely